docs/client_advanced.rst

```diff
@@ -640,3 +640,33 @@ are changed so that aiohttp itself can wait on the underlying
 connection to close. Please follow issue `#1925
 <https://github.com/aio-libs/aiohttp/issues/1925>`_ for the progress
 on this.
+
+
+Character Set Detection
+-----------------------
+
+If you encounter a :exc:`UnicodeDecodeError` when using :meth:`ClientResponse.text()`
+this may be because the response does not include the charset needed
+to decode the body.
```

Comment:
```suggestion
If you encounter an 'Automatic charset detection will be removed' warning
when using :meth:`ClientResponse.text()` this may be because the response
does not include the charset needed to decode the body.
```